assets/crx/js/embed.js

```diff
@@ -5,6 +5,13 @@ var webview = document.getElementById('webview');
 var findBox = document.getElementById('find-box');
 var findInput = document.getElementById('find-text');
 
+// Get dialog box
+var dialogBox = document.getElementById("dialog-box");
+var dialogText = dialogBox.getElementsByTagName('div')[0];
```

Comment:
Maybe we can use `dialogBox.querySelector();` and also for the `input` and two `button` elements?
https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelector